dev/dev.clj

```diff
@@ -21,8 +21,10 @@
             [planwise.system :as system]))
 
 ;; Logging configuration for development
-(timbre/merge-config! {:ns-blacklist ["com.zaxxer.hikari.*"]})
-(timbre/set-level! :info)
+(timbre/merge-config! {:level :debug
+                       :ns-blacklist ["com.zaxxer.hikari.*"
+                                      "org.apache.http.*"
+                                      "org.eclipse.jetty.*"]})
```

Comment:
`main` is not loaded during development. It's the entry point for production only.